draft-ietf-quic-ack-frequency.md

```diff
@@ -150,8 +150,10 @@ sending the following transport parameter ({{Section 7.2 of QUIC-TRANSPORT}}):
 min_ack_delay (0xff03de1a):
 
 : A variable-length integer representing the minimum amount of time in
-  microseconds by which the endpoint can delay an acknowledgement. This limit
-  could be based on the receiver's clock or timer granularity.
+  microseconds by which the endpoint that is sending this value is able to
+  delay an acknowledgement. This limit could be based on the receiver's clock
+  or timer granularity. This information is used by the peer for selecting a
+  suitable value in the Request Max Ack Delay field of the ACK_FREQUENCY frame.
```

Comment:
```suggestion
  or timer granularity. 'min_ack_delay' is used by the peer to avoid requesting too
  small a value in the Request Max Ack Delay field of the ACK_FREQUENCY frame.
```